.github/workflows/ci.yml

```diff
@@ -23,6 +23,22 @@ jobs:
     - name: Run tests
       run: |
         docker run rp
+        ret_rp=$?
+        echo "rp   : $ret_rp"
+        # docker run flux
```

Comment:
Looks like the `docker run`s are commented out?  Maybe a leftover from debugging?

Reply:
That's on purpose actually - the PR was only supposed to expand the RP tests and to document how multiple tests can be run, and I did not want to expand scope by adding those tests.  But, OTOH, if you think it makes sense I'd be happy to enable them - makes the PR cleaner also...  What do you think?

Reply:
Yeah, I'd vote for adding the `ret_X` around the various docker runs, but adding them around the `rp_parsl`, `rp_parsl_swift`, and `rp_parsl_swift_flux` images run below (it is the last component in the name whose CMD is run, so running `rp_parsl_swift` executes the swift tests, rp and parsl just happen to also be installed in the image.

Reply:
PS - Assuming that doesn't require any fixups in the other components' images/tests.  If it does, then I think this PR is fine as is.

Reply:
Enabling the tests do result in failures, specifically on `docker run flux`:
```
Unable to find image 'flux:latest' locally
docker: Error response from daemon: pull access denied for flux, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
Error: Process completed with exit code 125.
```
I don't mind using this PR to fix this, or to leave as is.  @SteVwonder : what would you prefer?

Reply:
We can leave as is.  Probably better to get this in sooner rather than later. Thanks @andre-merzky!